### ISSUE TITLE
how to modify/add/remove columns of a list?

### ISSUE THREAD

**Issue by taoeternal2018** [jasonrollins/shareplum#30]
Is it possible to modify a list with shareplum?

**Comment by Tibor2** [comment]
yes, have a look: https://shareplum.readthedocs.io/en/latest/tutorial.html

**Comment by lanceteo** [comment]
@Tibor2, I read the tutorial, but could not find anything relating to modifying/adding/removing columns. I might have missed it. Could you direct me to the section?

**Comment by jasonrollins** [comment]
@lanceteo I don't have the code written for adding or removing columns.  It would be under List.UpdateList().  I have some code there but it isn't working.
